source/manual/databases.html.md

```diff
@@ -0,0 +1,37 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: Databases on GOV.UK
+section: Infrastructure
+layout: manual_layout
+parent: "/manual.html"
+---
+
+Many GOV.UK applications use a database. See our [spreadsheet of applications and database engines](https://docs.google.com/spreadsheets/d/1rBZeeaT9XevaRvstMjw2YsbSsoB7IqgWjpJz5oAO9nM/edit#gid=1368371571) for details.
+
+## Database engines
+
+GOV.UK uses mostly PostgreSQL and some MySQL for apps that require a relational database. There's no particular reason why we use both, and the fact that we do is considered [tech debt](https://trello.com/c/zlfgSJlV/69-govuk-uses-mysql-and-postgresql-and-mongo).
+
+GOV.UK uses MongoDB for apps that require a document database. MongoDB is synonymous with DocumentDB; the name describes where how it is hosted, but the underlying engine is the same.
+
+Finally, GOV.UK uses an ElasticSearch database for search.
+
+## Hosting
+
+Instead of running these databases locally on the same instance as the application, they're hosted in separate infrastructure.
+
+Each Postgres and MySQL database runs in its own RDS instance. Whilst RDS instances are capable of hosting multiple databases, we decided to grant each database its own instance in [RFC-143](https://github.com/alphagov/govuk-rfcs/blob/main/rfc-143-split-database-instances.md).
+
+Mongo databases are hosted either in DocumentDB clusters (managed by AWS) or MongoDB clusters (managed by us on self-hosted EC2 instances). On production, there is currently one DocumentDB cluster for Licensify and one 'shared' DocumentDB cluster, each with three instances. There is also one Mongo cluster of three EC2s. We have agreed that [we should move apps from MongoDB to DocumentDB](https://docs.publishing.service.gov.uk/repos/govuk-aws/architecture/decisions/0038-mongo_replacement_by_documentdb.html).
+
+ElasticSearch is hosted on a large cluster of 'blue-search' EC2 instances.
```

Comment:
The `blue-search` instances run Search API. ElasticSearch is hosted in [AWS's OpenSearch service](https://eu-west-1.console.aws.amazon.com/esv3/home?region=eu-west-1#opensearch). It has two types of node - [master and data (or instance)](https://github.com/alphagov/govuk-aws/blob/main/terraform/projects/app-elasticsearch6/main.tf#L186-L193) that aren't visible in the EC2 management console.

Reply:
Thanks - I was definitely trying to piece some Search knowledge together from existing docs/assumptions and appreciate your corrections!

Reply:
Yep, totally appreciate your efforts! 👍